src/main/java/name/valery1707/kaitai/KaitaiGenerator.java

```diff
@@ -62,6 +68,10 @@ public KaitaiGenerator withSource(Iterable<Path> sources) throws KaitaiException
 		return this;
 	}
 
+	public KaitaiGenerator withSource(Path... sources) throws KaitaiException {
```

Comment:
\[Checkstyle\] WARNING: 'method def modifier' has incorrect indentation level 8, expected level should be 2\.